tests/test_plotting.py

```diff
@@ -0,0 +1,43 @@
+import shutil
+from pathlib import Path
+from nesta_ds_utils import plotting
+import pandas as pd
+import altair as alt
+import pytest
+
+
+def test_output_path():
```

Comment:
If you use the file_ops method I don't think you would need this test as it's already tested in test_file_ops